Jenkinsfile_nightly

```diff
@@ -0,0 +1,23 @@
+#!groovy
+
+properties([
+        pipelineTriggers([cron('00 21 * * *')]),
+        parameters([
+                string(name: 'URL_TO_TEST', defaultValue: 'https://cmc-citizen-frontend-aat.service.core-compute-aat.internal/', description: 'The URL you want to run these tests against'),
+        ])
+])
+
+@Library("Infrastructure")
+
+def product = "cmc"
+def component = "citizen-frontend"
+
+withNightlyPipeline("nodejs", product, component) {
```

Comment:
there is no slack notifications for this one too - but maybe this is correct :)

Reply:
I didnt want to set it up as it wasn't there before.